src/rust-2018/module-system/path-clarity.md

```diff
@@ -268,21 +203,21 @@ see their names, instead of having a bunch of tabs named `mod.rs`.
 
 # Uniform paths
 
-> Uniform paths are a nightly-only feature.
+![Minimum Rust version: 1.32](https://img.shields.io/badge/Minimum%20Rust%20Version-1.32-brightgreen.svg)
 
 The uniform paths variant of Rust 2018 simplifies and unifies path handling
 compared to Rust 2015. In Rust 2015, paths work differently in `use`
 declarations than they do elsewhere. In particular, paths in `use`
 declarations would always start from the crate root, while paths in other code
-implicitly started from the current module. Those differences didn't have any
+implicitly started from the current scope. Those differences didn't have any
 effect in the top-level module, which meant that everything would seem
 straightforward until working on a project large enough to have submodules.
 
 In the uniform paths variant of Rust 2018, paths in `use` declarations and in
-other code always work the same way, both in the top-level module and in any
-submodule. You can always use a relative path from the current module, a path
-starting from an external crate name, or a path starting with `crate`, `super`,
-or `self`.
+other code almost always work the same way, both in the top-level module and
```

Comment:
The "almost" bit here will leave the reader wondering...

Reply:
Maybe: "paths in `use` declarations and in other code work the same way, ...".

This is less "100% sure" but also doesn't say "maybe".